src/extern/bpf/btf.c

```diff
@@ -0,0 +1,3023 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
```

Comment:
there exists btf.c under src/extern/libbpf/src.

Reply:
If you look line by line, there are differences. I needed for batch update.